fw/hashseg.py

```diff
@@ -191,10 +191,62 @@ def pack(self):
 			+ self.signature + self.cert_chain
 
 
+@dataclass
+class HashSegmentV7(_HashSegment):
+	version: int = 7  # Header version number
+
+	common_metadata_size: int = 24 # This defaults to 24
+	metadata_size_qcom: int = 0  # Size of metadata from Qualcomm
+	metadata_size: int = 0	# Size of metadata from OEM
+	code_size: int = 0
+	signature_size_qcom: int = 0  # Size of signature from Qualcomm
+	cert_chain_size_qcom: int = 0  # Size of certificate chain from Qualcomm
+	signature_size: int = 0  # Size of attestation signature
+	cert_chain_size: int = 0  # Size of certificate chain
+	major: int = 0
+	minor: int = 0
+	sw_id: int = 0 # Software ID
+	sec_sw_id: int = 0
+	hash_table_algo: int = 3 # SHA384
+	measurement_register_target: int = 0
+
+	common_metadata = b'\x00' * 24 # structure unknown
```

Comment:
The way I understand it (from looking at the coreboot code), major+minor+sw_id+sec_sw_id+hash_table_algo+measurement_register_target _are_ the "common_metadata", and if each of them is 32-bit (4 bytes) then `6*4 = 24` it would fit. Do we have the common_metadata twice in here?